### ISSUE TITLE
Expand repository to have multiple files based on the curriculum outline.

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#4]
This will entail creating new files in this repository and moving the curriculum away from the README.md file.

**Comment by CheekeyMonkey** [comment]
I can help with this if needed. Might be best to make the Readme more of a syllabus/overview of the entire OSPAT.

**Comment by unshakenme** [comment]
Agreed it would be best for Readme to serve that purpose. Thanks for helping out!

**Comment by CheekeyMonkey** [comment]
Be good to delegate some tasks around. Is there anyone else willing to work on this?

**Comment by voyagermedical** [comment]
> Be good to delegate some tasks around. Is there anyone else willing to work on this?

Yes happy to, I have added a proposed basic syllabus in another thread.

**Comment by a-n-g-i-e-r-i** [comment]
I work professionally at a remote learning organization and one framework that has worked really well to organize curriculum is to organize content/files within directories based on a specific unit, and then add directory links to the README. There could also be a top level README with sub-READMEs. Another factor to consider is that while the curriculum itself remains hosted on GitHub to keep it version controlled and opensource, the delivery of the curriculum could be linked to in an LMS to create cohort "instances" which would aid in the delivery of the curriculum and managing learners if course delivery is something OSPAT might be interested in at some point. Imagine hosting the curriculum in a MOOC site for example, optimizing the structure of the GitHub repo for an eventuality like that might be a good use of time now.

**Comment by CheekeyMonkey** [comment]
Hey a-n-g-i-e-r-i, great comments. Might I suggest making a fork with your idea and then doing a submit request?

**Comment by unshakenme** [comment]
Approved. Thanks a-n-g-i-e-r-i!

**Comment by CheekeyMonkey** [comment]
Awesome thanks for contributing!

**Comment by a-n-g-i-e-r-i** [comment]
Really exciting to contribute to this project! I'll be at Horizons next week talking up OSPAT, btw. Besides this repo, are there any resources it would be helpful to point interested folks towards? I'm also a Fireside Project volunteer and will make an announcement on our team's Discord regarding the opportunity to contribute to OSPAT. @CheekeyMonkey @unshakenme

**Comment by unshakenme** [comment]
That’s wonderful, thanks Tristan! Hope we get a few more contributors from your efforts 🙏

**Comment by CheekeyMonkey** [comment]
The syllabus looks a lot better with the Headings etc

**Comment by CheekeyMonkey** [comment]
If you would segment, Id be happy to contribute to the update.

**Comment by a-n-g-i-e-r-i** [comment]
@CheekeyMonkey what do you mean by segment? I would really like to contribute further
